docs/release-notes/2.41.2.md

```diff
@@ -89,26 +89,41 @@ Read our [blog post](https://www.percona.com/blog/monitoring-a-postgresql-patron
 
-[PMM-12960](https://perconadev.atlassian.net/browse/PMM-12960) - [Tech Preview] The latest PMM release includes redesigned Kubernetes dashboards with improved clarity and usability.
 
 ## Fixed issues
 
-[PMM-12806](https://perconadev.atlassian.net/browse/PMM-12806) - In previous versions, PMM Server used certain non-configurable Victoria Metrics settings with default values. With this release, we are updating this behavior to enable you to modify default values for Victoria Metrics settings using environment variables. Previously, environment variables were overlooked as Victoria metrics prioritized specific command line flags passed by PMM over environment variables.
+- [PMM-12806](https://perconadev.atlassian.net/browse/PMM-12806) - In previous versions, PMM Server used certain non-configurable Victoria Metrics settings with default values. With this release, we are updating this behavior to enable you to modify default values for Victoria Metrics settings using environment variables. Previously, environment variables were overlooked as Victoria metrics prioritized specific command line flags passed by PMM over environment variables.
+
+- [PMM-12348](https://perconadev.atlassian.net/browse/PMM-12348) - The ClickHouse engine was updated from the ordinary engine to the Atomic engine, which is the most robust engine, after upgrading PMM to version 2.41.0.
+
+- [PMM-12785](https://perconadev.atlassian.net/browse/PMM-12785) - Fixed an issue where VictoriaMetrics scrape configuration contained unsupported parameters. 
+
+- [PMM-12805](https://perconadev.atlassian.net/browse/PMM-12805) - While monitoring MongoDB servers, the disk space was being consumed due to the `CommandNotSupportedOnView` message in the logs. This issue has now been resolved.
+
+- [PMM-12809](https://perconadev.atlassian.net/browse/PMM-12809) - Fixed several CVEs in PMM versions v2.40.1 and above.
+
+- [PMM-12852](https://perconadev.atlassian.net/browse/PMM-12852) - An internal server error was thrown while attempting to modify any setting in PMM. This issue has been resolved now.
+
+- [PMM-12986](https://perconadev.atlassian.net/browse/PMM-12986) - Fixed an issue related to the `PostgreSQL connections in use` template which was previously not functioning for Alerts.
 
-[PMM-12348](https://perconadev.atlassian.net/browse/PMM-12348) - The ClickHouse engine was updated from the ordinary engine to the Atomic engine, which is the most robust engine, after upgrading PMM to version 2.41.0.
+- [PMM-12894](https://perconadev.atlassian.net/browse/PMM-12894) - We have limited the number of connections used by postgresql_exporter. Additionally, we have fixed an issue where connections are closed as soon as they are no longer needed, thereby preventing any hanging or idle connections.
```

Comment:
it's an improvement that should improve the way PMM affects user environment(less CPU usage). And it's postgres_exporter.

Reply:
Fixed